Functions/xASL_bids_CompareStructures.m

```diff
@@ -136,15 +136,17 @@
 function reportTable = xASL_bids_CompareStructures_AddEntriesToTable(reportTable,results,datasetA,datasetB)
 
     % Go through missing folders
-    if size(results.(datasetA).missingFolders,1)>0 && ~isempty(results.(datasetA).missingFolders{1})
+    if numel(results.(datasetA).missingFolders)>0 && ...
+            ~isempty(results.(datasetA).missingFolders{1})
         for iElement=1:size(results.(datasetA).missingFolders,1)
             dataset = cellstr(datasetA);
             name = cellstr('Missing folder');
             message = cellstr(results.(datasetA).missingFolders{iElement,1});
```

Comment:
and here - why, iElement,1? A potential problem

Reply:
Fixed here: 4d00c8f 👍